README.md

```diff
@@ -1,5 +1,16 @@
 # fty-rest
 
+## How to build
+
+To build, run:
+
+```bash
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=usr -DBUILD_TESTING=On ..
```

Comment:
not sure, but -DCMAKE_INSTALL_PREFIX=/usr looks better, and at least it would work for local builds :)

Reply:
I missed this... :( Time to change docs. :)

Reply:
https://confluence-prod.tcc.etn.com/display/BiosWiki/Migrate+from+zproject+to+CMake